LivingSpace/Src/extra_globals.uci

```diff
@@ -0,0 +1,18 @@
+// Log helpers
+
+//`define PA_Trace(msg) `Log(`msg, !class'X2DownloadableContentInfo_PrototypeArmoury'.default.SuppressTraceLogs, 'PrototypeArmoury_Trace')
+
+`define LS_Log(msg) `Log(`msg,, 'LivingSpace')
+
+//`define PA_Warn(msg, bLog, bRedScreen) class'PAHelpers'.static.LogWarn(`msg, true, `bLog, `bRedScreen)
+//`define PA_WarnNoStack(msg, bLog, bRedScreen) class'PAHelpers'.static.LogWarn(`msg, false, `bLog, `bRedScreen)
```

Comment:
These commented lines should be removed to improve readability and prevent confusion

Reply:
I left them there for easy reference (if we ever want to implement them in LS), but you are probably right - we can easily look at other code anyway